tollway/utils.py

```diff
@@ -1,43 +1,70 @@
 import json
+import logging
+from collections import namedtuple
 from pathlib import Path
-from typing import Mapping, TypedDict, Union
+from typing import Callable, Mapping, TypedDict, Union
 
 from decouple import config
 from google.cloud import pubsub_v1
+from google.cloud.pubsub_v1.publisher.futures import Future
 from google.oauth2 import service_account
 
+Topic = namedtuple("Topic", ["publisher", "topic_path"])
```

Comment:
I prefer typed ones these days, see https://codechalleng.es/tips/define-a-namedtuple-with-type-hints

Reply:
yeah, this is better